__tests__/components/AnonymousChat/child/ChatItemMyTextV2.test.tsx

```diff
@@ -1,43 +1,178 @@
+    const contextValue = {
+      chat: [{replyTarget: null}, setReplyPreview],
+      profile: [{myProfile: {}}]
+    };
+
+    const {getByTestId} = render(
+      <Context.Provider value={contextValue}>
+        <ChatItemMyTextV2
+          avatar={avatar}
+          username="username"
+          isContinuous={false}
+          message="message"
+          time="time"
+          chatType="SIGNED"
+          messageType="regular"
+          data={{}}
+        />
+      </Context.Provider>
     );
 
-    expect(toJSON()).toMatchSnapshot();
+    act(() => {
+      fireEvent(getByTestId('swipeable'), 'onSwipeableOpen', 'left');
+    });
+
+    expect(setReplyPreview).toHaveBeenCalled();
   });
 
-  it('RENDER continuous should match snapshot', () => {
-    const {toJSON} = render(
-      <ChatItemMyTextV2
-        avatar={avatar}
-        username="username"
-        isContinuous={true}
-        message="message"
-        time="time"
-      />
+  it('should display the username and time when isContinuous is false', () => {
+    const contextValue = {
+      chat: [{replyTarget: null}, jest.fn()],
+      profile: [{myProfile: {}}]
+    };
+
+    const {getByText} = render(
+      <Context.Provider value={contextValue}>
+        <ChatItemMyTextV2
+          avatar={avatar}
+          username="username"
+          isContinuous={false}
+          message="message"
+          time="time"
+          chatType="SIGNED"
+          messageType="regular"
+          data={{}}
+        />
+      </Context.Provider>
     );
 
-    expect(toJSON()).toMatchSnapshot();
+    expect(getByText('username')).toBeTruthy();
+    expect(getByText('time')).toBeTruthy();
+  });
+
+  it('should not display the username and time when isContinuous is true', () => {
+    const contextValue = {
+      chat: [{replyTarget: null}, jest.fn()],
+      profile: [{myProfile: {}}]
+    };
+
+    const {queryByText} = render(
+      <Context.Provider value={contextValue}>
+        <ChatItemMyTextV2
+          avatar={avatar}
+          username="username"
+          isContinuous={true}
+          message="message"
+          time="time"
+          chatType="SIGNED"
+          messageType="regular"
+          data={{}}
+        />
+      </Context.Provider>
+    );
+
+    expect(queryByText('username')).toBeNull();
+    expect(queryByText('time')).toBeNull();
+  });
+
+  it('should display the message', () => {
+    const contextValue = {
+      chat: [{replyTarget: null}, jest.fn()],
+      profile: [{myProfile: {}}]
+    };
+
+    const {getByText} = render(
+      <Context.Provider value={contextValue}>
+        <ChatItemMyTextV2
+          avatar={avatar}
+          username="username"
+          isContinuous={false}
+          message="message"
+          time="time"
+          chatType="SIGNED"
+          messageType="regular"
+          data={{}}
+        />
+      </Context.Provider>
+    );
+
+    expect(getByText('message')).toBeTruthy();
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This test case checks if the message is displayed. However, it does not check if the message is displayed correctly when `messageType` is not "regular". It would be more comprehensive to also test other message types.



<!-- This is an auto-generated comment by OSS CodeRabbit -->